smartcontractkit/external-adapters-js
# Refactored adapter endpoint names to support new forex endpoints

## Closes #12087

## Description
Refactored adapter endpoint names to support new forex endpoints.
......

## Changes
Changed supportedEndpoints and READMEs



## Steps to Test

1. Ensure that the data is the same between the old and new endpoints
2. Make sure default endpoints work

## Quality Assurance
